docs/tutorials/tutorials.md

```diff
@@ -0,0 +1,382 @@
+| All Upper Case      | &#10003;                   |   &#10003;                   | &#10003; (with warning msg) | `specified`           |
+| All Lower Case      | &#10003;                   |   &#10003; (with warning msg)| &#10003; (with error);      | `specified`           |
+| Upper and Lower Case| &#10003;                   |   &#10003; (with warning msg)| &#10003;                    | `specified`           |
+| Anonymous Group Name| &#10003; (with error)      |   &#10003;                   | &#10003; (with error)       | `any`                 |
+
+#### Keyword `exists`
+
+The `exists` keyword plays a pivotal role in defining the optionality of NeXus concepts `attribute`, `field`, `link`, and `group`. It provides crucial insights for the expected presence or absence of these concepts within the NeXus data structure when implementing the NeXus definitions in NeXus files. By default, all the concepts of a base class are optional, while in an application definition, all concepts are required.
+
+
+Currently, the accepted values for the `exists` keyword encompass:
+`required`: Indicates that the NeXus concept must be present within the structure. Any validation of a NeXus file will fail or give a warning if required concepts (for a given application definition) are not available.
+
+`recommended`: Suggests that the NeXus concept is advised, but not mandatory.
+`optional`: Denotes that the NeXus concept is not mandatory and may be absent.
+
+`[min, <number>, max, <number> or infty]`: Represents an array type value that signifies the multiplicity of the NeXus concepts. For instance, a concept having the keyword `exists: [min, 3, max, infty]` implies that this concept must come with a minimum of three instances and may extend to any number of instances.
+
+**`exists` in YAML**
+
+=== "YAML"
+    ```yaml
+    transmission_correction(NXcalibration):
+      exists: optional
+      doc: |
+        This calibration procedure is used to account for the different transmission efficiencies.
+    ```
+=== "XML"
+    ```xml
+    <group type="NXcalibration" optional="true">
+      <doc>
+        This calibration procedure is used to account for the different transmission efficiencies.
+      </doc>
+    </group>
+    ```
+
+In the above example the group `transmission_correction` is an optional group.
+
+#### Keyword `unit`
+The `unit` keyword is used to define the NeXus-compliant unit categories.
+
+**`unit` in YAML**
+
+=== "YAML"
+    ```yaml
+    detector_voltage(NX_FLOAT):
+      unit: NX_VOLTAGE
+      doc: |
+        Voltage applied to detector.
+    ```
+=== "XML"
+    ```xml
+    <field name="detector_voltage" type="NX_FLOAT">
+      <unit>NX_VOLTAGE</unit>
+      <doc>Voltage applied to detector.</doc>
+    </field>
+    ```
+In the above example, the `detector_voltage` field is defined as a `NX_FLOAT` type with a unit of `NX_VOLTAGE`, indicating that the values stored in this field are measured in volts or millivolts or other voltage units.
```

Comment:
Do not point that it is NX_FLOAT in this paragraph important is the unit type only

Reply:
I really not see any problem mentioning type `NX_FLOAT`, but the overall paragraph is mainly focused on the `units`.